azkaban-common/src/main/java/azkaban/imagemgmt/models/ImageRampRule.java

```diff
@@ -0,0 +1,50 @@
+package azkaban.imagemgmt.models;
+
+import java.util.List;
+import java.util.Set;
+
+
+public class ImageRampRule extends BaseModel {
+  // ramp rule name
+  String ruleId;
+  // image name of ramp rule for
+  String imageName;
+  // image version of ramp rule for
+  String imageVersion;
+  // rule owners
+  Set<String> owners;
```

Comment:
From the code logic, owners here should always == image owner? Introducing too many and redundant owner names will make future maintenance too complicated. Can we just discard this field but rely on image_ownership table to tell the ownership?
What's your thought?

Reply:
This is likely a subset of owners, however, I agree that image_ownership is better bet. It keeps single source of truth.

Reply:
Sorry to make the confusion here but I meant to add HP flow owners as rule owners too. The owner source of truth would come from image owners for regular rule and hp flow owners for HP flow rule.
This PR is only for regular rule creation, So I didn't add HP flow part in this one. The next one would add HP as owers

Reply:
I see. It makes sense.
But I feel like owners field for non-hp rule is still not good. We had so many asks before to add some member to the owner list. If we want to store owner list also in this table, it means whenever an ask comes, we need to add owner in two places (or maybe more places b/c one imageName may have multiple ImageRampRules): higher maintenance cost.

Reply:
Initially I do added such logic to only add group owners into the owner list. But I wonder if this is too harsh? Because many teams are missing the group as owner, and we likely get little group as owner, we can ask them to update the image ownerships with their group, but then it means we need to open such API to change. Because all the other APIs are supposed to rely on this owner column to operate the permission.
I can add it back if you think we should do that. Let me know how you feel.

Reply:
Or maybe a safe second check would just work. If update/delete API failed to authorize, then just do another permission check.

Reply:
The current logic makes sense. Thanks for discussing with me in offline!